module/Photo/config/module.config.php

```diff
@@ -145,6 +145,32 @@
                             ],
                         ],
                     ],
+                    'setprofilephoto' => [
```

Comment:
I believe `set_profile_photo` would be a better name for this if you want it to match with our existing naming scheme.